src/podman.scss

```diff
@@ -236,3 +257,6 @@
     font-weight: 400;
 }
 
+#registry-select option:first-child {
```

Comment:
This is specific to  src/ImageSearchModal.css  and should be moved there.

Reply:
Great point.

I added CSS where it already existed, or when I made a mistake. :grin:  But it should be moved.

Reply:
Ah, that's from @marusak, a week ago. But it doesn't work in anything but Blink. Non-Chromium-based browsers just ignore styles on options.

Reply:
I cannot find any good reference which browsers support what CSS for `<select>`'s `<option>`s, but I do know from experience that browsers are generally unhappy to mess with default platform styling for the popup. I _think_ it's a security-related problem, as the select's popup is rendered at OS level (above the page) and modifying it (which _used to be possible_, years ago) meant that people _could_ hijack it with CSS to make impostor widgets which could appear outside of the page. This is why even Chrome only supports a strict subset of CSS on `<options>`... and other browsers basically disable it outright.

As I cannot find a reference to this, I could be misremembering or making it up. :shrug: But I'm pretty sure this is the case.

Anyway:

Summary: Not supported in anything but Chrome. Should be in ` src/ImageSearchModal.css` as @KKoukiou states, or it should be dropped (as it's Chrome-only).